### ISSUE TITLE
population mutation spectra?

### ISSUE THREAD

**Issue by goldmich** [harrispopgen/mutyper#8]
Would be awesome to output the mutation spectrum for a group of individuals - effectively colsums of a ksfs, i.e. input data for the Harris & Pritchard 2017 heatmaps. Thanks!

**Comment by WSDeWitt** [comment]
@goldmich Here's a workaround to get this on the command line:
```bash
mutyper variants <fasta> <vcf> | mutyper ksfs - | cut -f2- | awk 'NR==1{print $0} {for (i=1;i<=NF;i++) sum[i]+=$i} END{for (i in sum) printf sum[i] "\t"; print }'
```
Explanation:
- the `cut` command removes the sample_frequency column from the `ksfs` output
- the `awk command prints the mutation types, then the `ksfs` row sums for each

**Comment by WSDeWitt** [comment]
closed in 17d82cf
